client/src/components/OpenAnswerForm.js

```diff
@@ -5,16 +5,19 @@ import { Form, Button, Grid } from "semantic-ui-react";
 class OpenAnswerForm extends React.Component {
   state = { name: "", qType: "", explanation: "" };
 
-  handleChange = (e, { name, value, }) => this.setState({ [name]: value,});
+  handleChange = (e, { name, value, }) => this.setState({ [name]: value, });
 
   handeSubmit = e => {
     e.preventDefault();
     const { quiz_id } = this.props;
     const question = { ...this.state, quiz_id: quiz_id, qType: "open" };
-    axios.post(`/api/quizzes/${quiz_id}/questions`, question).then(res => {
-      this.props.addQuestion(res.data, true);
-      console.log(res);
-    });
+
```

Comment:
Can you change some of this formatting so that it is indented with the same functions? I Know prettier can make it janky some times.